### ISSUE TITLE
why the loss_cls and loss_bbox are zero during the train when i use yolox in mmdet 3.0.0?

### ISSUE THREAD

**Issue by VincentMy** [open-mmlab/mmdetection#10338]
when i use  yolox model to train my  own data in mmdet 3.0.0 version ,the losses are  zero all the time
the model is yolox_s_8xb8-300e_coco.py,here is my config:
![image](https://github.com/open-mmlab/mmdetection/assets/47235130/6f317f22-44c9-4086-820e-44fc5bde9c9f)
![image](https://github.com/open-mmlab/mmdetection/assets/47235130/2d6e8736-a6c4-4f72-9d68-dc3d483bc6fd)
![image](https://github.com/open-mmlab/mmdetection/assets/47235130/d2f79440-e611-4546-bbeb-74dd21abff8c)
![image](https://github.com/open-mmlab/mmdetection/assets/47235130/197656d4-6b85-4aa6-97e1-27a152fb9e12)
![image](https://github.com/open-mmlab/mmdetection/assets/47235130/63a3c69b-9201-46ed-b109-97279fdfa608)
![image](https://github.com/open-mmlab/mmdetection/assets/47235130/d3e6a926-a5aa-469a-8b34-02298b6f1b6b)
![image](https://github.com/open-mmlab/mmdetection/assets/47235130/49815e7c-8f39-46b8-b960-de12e2e4898b)
![image](https://github.com/open-mmlab/mmdetection/assets/47235130/fc2b5485-cfd9-4eaa-9782-f1a810eb4617)

i  modified my config after read this guidance document https://mmdetection.readthedocs.io/en/latest/user_guides/train.html#train-with-customized-datasets
I seriously suspect that the category of images is wrong and  the way i did load the pretrained model is incorrect from the guidance document

the loss_cls and loss_bbox are zero and have no evaluate result
`05/16 18:01:05 - mmengine - INFO - Epoch(train)  [21][26/26]  lr: 9.9267e-03  eta: 17:11:18  time: 8.3338  data_time: 8.1844  memory: 5221  loss: 0.0013  loss_cls: 0.0000  loss_bbox: 0.0000  loss_obj: 0.0013`

`05/16 18:34:10 - mmengine - INFO - Saving checkpoint at 30 epochs
05/16 18:34:15 - mmengine - INFO - Evaluating bbox...
Loading and preparing results...
05/16 18:34:15 - mmengine - ERROR - /mc39/lib/python3.9/site-packages/mmdet/evaluation/metrics/coco_metric.py - compute_metrics - 461 - The testing results of the whole dataset is empty.
`

**Comment by VincentMy** [comment]
the loss_cls and loss_bbox are zero and have no evaluate result

**Comment by cosimo17** [comment]
@VincentMy  How to fix this? I also meet this problem.

**Comment by Time-Travel-x** [comment]
@VincentMy How to fix this? I also meet this problem.

**Comment by blaoke** [comment]
[BUG] meet too!!

**Comment by morningbearscau** [comment]
The same problem

**Comment by cosimo17** [comment]
I fixed this by correcting the "classes" in the config. In my previous setting, some class name in config file does not match the name in annotation.

**Comment by maichm** [comment]
solved! you can refer to https://mmdetection.readthedocs.io/en/latest/user_guides/train.html#prepare-a-config .
modify the variable "metainfo" as you own dataset classes in your configure file.
finally remeber to pack it to the "dataset", the attribute of train_dataloader and val_dataloader .
